doc/man1/openssl-version.pod.in

```diff
@@ -77,8 +78,29 @@ The random number generator source settings.
 
 The OpenSSL CPU settings info.
 
+=item B<-w>
+
+The OpenSSL B<OSSL_WINCTX> build time variable, if set.
+Used for computing Windows registry key names
+
 =back
 
+=head1 HISTORY
+
+In OpenSSL versions prior to 3.4, OpenSSL had a limitation regarding the
+B<OPENSSLDIR>, B<MODULESDIR> and B<ENGINESDIR> build time macros.  These macros
+were defined at build time, and represented filesystem paths.  This is common
+practice on unix like systems, as there was an expectation that a given build
+would be installed to a pre-determined location.  On Windows however, there is
+no such expectation, as libraries can be installed to arbitrary locations.
+B<OSSL_WINCTX> was introduced as a new build time variable to define a set of
+registry keys identified by the name openssl-<version>-<tag>, in which the
+<version> value is derived from the version string in the openssl source, and
+the <ctx> tag is derived from the B<OSSL_WINCTX> variable.  The values of
```

Comment:
Above you refer to this as `<tag>` but here you call it the `<ctx> tag` which seems inconsistent